frontend/lib/src/components/elements/GraphVizChart/GraphVizChart.tsx

```diff
@@ -38,75 +38,68 @@ interface Dimensions {
 const dummyGraphviz = graphviz
 dummyGraphviz // eslint-disable-line @typescript-eslint/no-unused-expressions
 
+const isFullScreen = (height = 0): boolean => Boolean(height)
+
 export function GraphVizChart({
   width: propWidth,
   element,
   height: propHeight,
 }: GraphVizChartProps): ReactElement {
   const chartId = `graphviz-chart-${element.elementId}`
 
+  const isFull = isFullScreen(propHeight)
+  const isFullRef = useRef(isFull)
+  // Update isFull state whenever propHeight changes
+  useEffect(() => {
+    isFullRef.current = isFull
+  }, [isFull])
+
   let originalHeight = 0
   let originalWidth = 0
 
-  const getChartData = (): string => {
-    return element.spec
-  }
+  const setSvgDimensions = useMemo(() => {
+    return (node: SVGGraphicsElement): void => {
+      const bbox = node.getBBox()
+      // eslint-disable-next-line react-hooks/exhaustive-deps
+      originalHeight = Math.round(bbox.height)
```

Comment:
Just curious, why is the `Math.round` necessary now vs. before just using `node.getBBox().height`

Reply:
When d3 initializes the SVG, the width and height are set as integer pt values. When getting the width using `node.getBBox()`, the width and height values are floats. It has always been necessary, but was not previously identified as such because the width and height calculation have been buggy and unused.